src/tests/components/LocationDetails.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import LocationDetails from "../components/LocationDetails";
+
+describe("LocationDetails", () => {
+  it("renders the correct cit and location props", () => {
+    const { getByText } = render(
+      <LocationDetails city="Manchester" country="UK" />
+    );
+
+    expect(getByText("Manchester, UK")).toBeInstanceOf(HTMLHeadingElement);
+  });
+});
```

Comment:
Theres scope for another test to be added here. What should be displayed in the event of an error message being passed into the component as a prop? We could fix this like so:
```
it("shows error message if passed in", () => {
  const { getByText } = render(
    <LocationDetails
      city="Manchester"
      country="UK"
      errorMessage="test error message"
    />
  );

  const element = getByText("test error message");
  expect(element).toBeInTheDocument();
  expect(element).toBeInstanceOf(HTMLHeadingElement);
});
```